security-groups/weblogic-ndelius.tf

```diff
@@ -80,6 +80,26 @@ resource "aws_security_group_rule" "ndelius_public_subnet_ingress_tls" {
   description       = "Public subnet in (TLS)"
 }
 
+resource "aws_security_group_rule" "ndelius_lb_self_ingress" {
+  security_group_id         = "${aws_security_group.weblogic_ndelius_lb.id}"
+  type                      = "ingress"
+  protocol                  = "tcp"
+  from_port                 = "80"
+  to_port                   = "80"
+  self                      = true
```

Comment:
do LBs need to have access to themselves?

Reply:
this is to support inter-LB communication, we have the same SGs applied to the HAProxy LB and the ALB so this allows them to talk to each other

Reply:
ahh ok that makes sense now!